back/app/models/permission.rb

```diff
@@ -29,6 +29,7 @@ class Permission < ApplicationRecord
     'ideation' => %w[posting_idea commenting_idea reacting_idea attending_event],
     'proposals' => %w[posting_idea commenting_idea reacting_idea attending_event],
     'native_survey' => %w[posting_idea attending_event],
+    'community_monitor_survey' => %w[posting_idea attending_event],
```

Comment:
Probably don't need attending event, but all other phases have attending event and there would be some refactoring not to have it